scaffold/cmd/templates/source/resources/plugin/client.go.tpl

```diff
@@ -0,0 +1,88 @@
+package plugin
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/cloudquery/plugin-sdk/v4/message"
+	"github.com/cloudquery/plugin-sdk/v4/plugin"
+	"github.com/cloudquery/plugin-sdk/v4/scheduler"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+	"github.com/rs/zerolog"
+	"github.com/{{.Org}}/cq-source-{{.Name}}/client"
+	"github.com/{{.Org}}/cq-source-{{.Name}}/resources/services"
+)
+
+type Client struct {
+	logger    zerolog.Logger
+	config    client.Spec
+	tables    schema.Tables
+	syncClient *client.Client
+	scheduler *scheduler.Scheduler
+
+	plugin.UnimplementedDestination
+}
+
+func New(ctx context.Context, logger zerolog.Logger, spec []byte, opts plugin.NewClientOptions) (plugin.Client, error) {
```

Comment:
Maybe call this `Configure` so it doesn't get confused with `client.New`? Currently we have 13 `New*` vs. 21 `Configure` in the monorepo source plugins.